packages/tasks/src/model-libraries.ts

```diff
@@ -980,6 +980,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path_extension:"pt" OR path_extension:"safetensors"`,
 		snippets: snippets.ultralytics,
 	},
+	zonos: {
```

Comment:
Can you tag all Zonos models with `library_name: zonos` in the model card metadata so that https://huggingface.co/models?other=zonos is populated?

If you don't own the repos yourself, you can always open PRs to let the authors know.